src/components/Popup/Popup.scss

```diff
@@ -151,6 +84,72 @@ $transition-distance: 10px;
             transform: rotate(-90deg);
         }
     }
+
+    &__content {
+        // Can be overridden, see Tooltip
+        --yc-popup-background-color: var(--yc-color-base-float);
+        --yc-popup-border-color: var(--yc-color-line-solid);
+        --yc-popup-border-width: 1px;
```

Comment:
These variables should remain on the root node

Reply:
I've moved it up to #{$block}